cpu/cc26x0/periph/rfc.c

```diff
@@ -76,35 +102,36 @@ void rfc_beacon(void)
         printf("%.2x", ((uint8_t *) rop)[i]);
     printf("\n");
 
-    run_command_ptr(&rop->op);
-}
+    //run_command_ptr(&rop->op);
+}*/
 
 void rfc_prepare(void)
 {
+    printf("===> %s <===\n",__FUNCTION__);
+    /* RFC POWER DOMAIN CLOCK GATE */
     PRCM->RFCCLKG = 1;
 
     PRCM->CLKLOADCTL = CLKLOADCTL_LOAD;
     while (!(PRCM->CLKLOADCTL & CLKLOADCTL_LOADDONE)) ;
 
-    PRCM->PDCTL0RFC = 1;
-    PRCM->PDCTL1RFC = 1;
+    /* RFC POWER DOMAIN */
+    PRCM->PDCTL0RFC = PDCTL0_RFC_ON;;
```

Comment:
For the sake of consistency I decide to use PDCTL0 and PDCTL1 rather than their respective alias register. Also this way we don't need additional definition to do essentially the same thing.